src/rules/at-rule-blacklist/README.md

```diff
@@ -0,0 +1,49 @@
+# at-rule-blacklist
+
+Specify a blacklist of disallowed at-rules.
+
+```scss
```

Comment:
Can you write as 

```css
    @keyframes name {}
/** ↑
 * At-rules like this */
```

i.e. Align everything as far to the left as it'll go, [as such](https://github.com/stylelint/stylelint/tree/master/src/rules/function-blacklist#function-blacklist)

And as @evilebottnawi said, please use `css` fences rather than `scss`.